inst/htmljs/animint.js

```diff
@@ -1766,6 +1771,17 @@ var animint = function (to_select, json_file) {
       // event will be fired on the original input.
       selectized_array[v_name].setValue(selected_ids, true);
     }
+
+    // For each updated geom, check if the axes of the plot need to be
+    // updated and update them
+    s_info.update.forEach(function(g_name){
+      var plot_name = g_name.split("_").pop();
+      var axes = Plots[plot_name]["options"]["update_axes"];
+      if(axes != null){
+        update_scales(plot_name, g_name, axes, v_name, value);
```

Comment:
hey thanks for sharing this first implementation

I find it strange that `update_scales` takes `g_name` as a parameter. Why do you need call it for every geom?

I would think that after a click to change a selection variables, we only need to update each axis once.